tests/src/readytorun/main.cs

```diff
@@ -49,8 +50,45 @@ public static string OpenClosedDelegateTarget(this string x, string foo)
     }
 }
 
+
+public enum Test
+{
+    Foo,
+    Bar
+}
+
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+public class TestAttribute : Attribute
+{
+    public TestAttribute(Test[] valuesOne, Test[] valuesTwo, Test[] valuesThree, Test[] valuesFour)
+    {
+    }
+}
+
 class Program
 {
+    static void TestCtorAttr()
+    {
+        var type = typeof(Program);
+        var method = type.GetMethod("Main", BindingFlags.Static | BindingFlags.NonPublic);
+        var attributeEn = method.CustomAttributes.GetEnumerator();
+        attributeEn.MoveNext();
+        var attribute = attributeEn.Current.ConstructorArguments;
```

Comment:
"constructorArgument" (or "argument") would be a more descriptive name than "attribute"